KubeArmor/common/common.go

```diff
@@ -329,9 +332,15 @@ func GetExternalIPAddr() string {
 
 // IsK8sLocal Function
 func IsK8sLocal() bool {
+	if !IsK8SEnv {
```

Comment:
```suggestion
	if !cfg.K8sEnv {
```

Reply:
It is not possible to include config in common as it causes circular dependency. Hence this global variable.